server/services/nl.py

```diff
@@ -120,11 +234,38 @@
 }
 
 
+def _add_to_set_from_list(set_strings, list_string):
+  for v_str in list_string:
+    if type(v_str) == str:
+      # Only add words which are strings.
+      set_strings.add(v_str.lower())
+
+
+# Use QUERY_CLASSIFICATION_HEURISTICS to add all words to this dict.
+STOP_WORDS_QUERY_CLASSIFICATION = set()
+for (_, v) in QUERY_CLASSIFICATION_HEURISTICS.items():
+  if isinstance(v, list):
+    # If 'v' is a list, add all the words.
```

Comment:
Some of the entries of v might be multiple word phrases. Either we should rework the entries of v to be one word only, or we should add logic to remove multi word phrases first

Reply:
Good point! After the refactoring, I am now inserting words after splitting.